tailwind/theme-tailwind-variables.scss

```diff
@@ -0,0 +1,106 @@
+$--tw-transparent: transparent;
```

Comment:
hrmm, should this file be gitignored since it would be built on every webpack run? Might not want to trigger any GrapeVine changes on every tw color change.